charts/config-server/templates/_overides.tpl

```diff
@@ -442,6 +442,11 @@ All env variables that are accessed from mosip config properties
     secretKeyRef:
       name: {{ .Values.overrides.secrets.confSecretsVarious }}
       key: mosip-ida-kyc-token-secret
+- name: SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_MIMOTO_WALLET_BINDING_PARTNER_API_KE
```

Comment:
Y is missing

SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_MIMOTO_WALLET_BINDING_PARTNER_API_KE